### DISCUSSION TITLE
9월 6일 일기

### DISCUSSION THREAD

**Discussion by iamyanghee** [Melon-Shake/API#78] [team]
오늘 일정
- 15시, 중간보고
- 18시, 멘토링

**Comment by iamyanghee** [comment]
이번주 테마는 적재적재 디비에 다 쌓아야지
오늘은 spotify 3종이랑 챠트 6종 다 쌓아야지

그리고 가보고 싶었던 구스아일랜드 가서 저녁 먹어야지 얌얌

**Comment by jungssg** [comment]
아침엔 너무 졸리다😪
